gist/gist.py

```diff
@@ -138,19 +138,6 @@ def __init__(self, token, editor=None):
         """
         self.token = token
         self.editor = editor
-        self.proxies = {}
```

Comment:
It does indeed pull them from the environment. My environment requires all access occur through a proxy and have the HTTP_PROXY, https_proxy, and REQUESTS_CA_BUNDLE environment variables set appropriately. Your code (as well as everything else that uses requests) works fine without any specific session configuration.

 If I use the module in its current state, it does use the proxy but I get a SSL error due to requests not trusting the private cert that the proxy uses. Apparently manually setting the proxy as you are doing bypasses the built-in logic that pulls configuration from the environment.